packages/core/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extends": "../../tsconfig.base.json",
+    "extends": "../../tsconfig.json",
```

Comment:
What's the reason for the rename?

Reply:
Forgot to remove it from the v2.6.1 lint upgrade, will change it back